pgcli/main.py

```diff
@@ -383,6 +379,18 @@ def initialize_logging(self):
         pgspecial_logger.addHandler(handler)
         pgspecial_logger.setLevel(log_level)
 
+    def initialize_keyring(self):
+        self.keyring = None
+
+        keyring_enabled = self.config["main"].as_bool("keyring")
+        if keyring_enabled:
+            # Try best to load keyring (issue #1041).
+            import importlib
+            try:
+                self.keyring = importlib.import_module('keyring')
+            except Exception:  # ImportError for Python 2, ModuleNotFoundError for Python 3
+                pass
```

Comment:
A `logger.debug` may be useful here. In case someone wants to use keyring, and it won't load, and we want to see why.

Reply:
Sure, I will add it in next revision.